tendrl/commons/objects/definition/master.yaml

```diff
@@ -906,11 +910,11 @@ namespace.tendrl:
           enabled: true
```

Comment:
change the name to plural,  "CheckClusterNodesUp" and move this atom under NS.tendrl.objects.cluster.atoms.CheckClusterNodesUp

Reply:
i will change it